.env.sample

```diff
@@ -0,0 +1,6 @@
+DB_USER=
```

Comment:
Let's kindly add values to this, just some sample values to serve as a guide. 

I'm thinking in most cases the likely ones to be changed will be the user and password,

PS: Usually, its just the `env.sample` that is copied/duplicated to serve as the .env so might help

Reply:
ok noted. Thanks